### PR TITLE
[INLONG-620]Selector for Go SDK

### PR DESCRIPTION
**Pull request by TszKitLo40**

### CODE REVIEW

**Comment by TszKitLo40** [comment]
@gosonzhang @charlely PTAL

File: tubemq-client-twins/tubemq-client-go/selector/ip_selector.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+)
+
+func init() {
+	s := &ipSelector{}
+	s.indexes = make(map[string]int)
+	Register("ip", s)
+	Register("dns", s)
+}
+
+type ipSelector struct {
```
Comment (charlely) [inline tubemq-client-twins/tubemq-client-go/selector/ip_selector.go:32]: Can the following structure be used?
type ipSelector struct {	
indexes map[string]ipServices
}

type  ipServices struct {
nextIndex int
addresss []string
}

Comment (TszKitLo40) [reply tubemq-client-twins/tubemq-client-go/selector/ip_selector.go:32]: Good idea.

**Review by charlely — APPROVED** [review]
LGTM

**Review by charlely — APPROVED** [review]
LGTM

**Comment by gosonzhang** [comment]
+1

**Pull request merged**